docker-compose.yml

```diff
@@ -51,27 +51,52 @@ services:
       credentials:
         condition: service_completed_successfully
     healthcheck:
-      test:
-        [
-          "CMD",
-          "sh",
-          "-c",
-          "kill -0 $$(cat /tmp/server.pid)"
-        ]
+      test: [ "CMD", "sh", "-c", "kill -0 $$(cat /tmp/server.pid)" ]
       interval: 5s
       timeout: 10s
       retries: 20
     image: python:3.10-slim
     networks:
       - public
     ports:
-      - "8080:8080"
+      - "8081:8081"
     restart: always
     volumes:
       - ./servers/oidc.py:/oidc.py
       - shared:/shared
 ###
 
+  auth:
+    container_name: auth
+    profiles:
+      - auth
```

Comment:
```suggestion
```
Auth shouldn't need to be part of a profile

Reply:
Indeed.